draft-jennings-moq-secure-objects.md

```diff
@@ -201,326 +190,248 @@ name but different data.
 One of the ways system keep the object names unique is by using a fully
 qualified domain names or UUIDs as part of the TrackNamespace.
 
+# Secure Objects
+
+A Secure MOQT Object is MOQT OBJECT_STREAM or OBJECT_DATAGRAM messages that has
+been protected with the scheme defined in this draft, so that its payload is
+encrypted and the whole message is authenticated by an authentication tag in the
+payload:
+
+~~~ aasvg
+   OBJECT_STREAM Message {
+ +-----------------------------------------+
+ |   Subscribe ID (i)        authenticated |
```

Comment:
Given the contention between track and subscribe id , I would rather just show all the data in the header that is not needed as just sort of ... and make this so we did not need to keep editing as base spec changed